static/js/script.js

```diff
@@ -66,7 +66,14 @@ function uploadFiles(event){
                     last_user_index = chat_user_index;
                 }
             } else {
-                show_error_message(response.error_message);
+                $('.upload-popup').prepend('<div class="alert alert-danger alert-dismissible">' +
+                '<button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>' +
+                response.error_message + 
+                '</div>');
+
+                setTimeout(function() {
+                    $('.alert').fadeOut('slow');
+                }, 4000);
```

Comment:
Why did you add this here? Can you see the problem with `show_error_message` function, since there are other scenarios as well which can raise error.

Reply:
The thing is in index.html, you passed in a reference to displace an error message if a message is set or not.

However, you are using Jinja syntax to determine that and not pure JavaScript, so in reality, the error is only set if it is passed in from app.py and not when there is an error with asynchronous API calls by JavaScript

Reply:
There's a method in `script.js` named `show_error_message`, you can fix/modify that method instead of adding the code here

Reply:
Ok. I'll have a look